gm4_calling_bell/data/gm4_calling_bell/functions/tick.mcfunction

```diff
@@ -0,0 +1,4 @@
+#
```

Comment:
Either don't put a comment here or populate it with the usual header comment info.